src/test/resources/features/FraudCRIAPI.feature

```diff
@@ -3,6 +3,6 @@ Feature: Fraud CRI API
 
   @intialJWT_happy_path @build
   Scenario: Acquire initial JWT (STUB)
```

Comment:
We have this test marked with the build tag so I think the the argument for each should be the build environment

Reply:
I have added 2 tests one with tag @dev and another with tag @build